packages/opds-web-client/src/app.tsx

```diff
@@ -10,6 +10,37 @@ import { State } from "./state";
 import AuthPlugin from "./AuthPlugin";
 import "./stylesheets/app.scss";
 
+const OPDSCatalogRouterHandler = (config) => {
+  interface OPDSCatalogProps {
+    params?: any;
```

Comment:
Isn’t this type just collection url and book url?

Reply:
Yes, updated it.